recipes/topas/build.sh

```diff
@@ -4,7 +4,7 @@ set -eu -o pipefail
 outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
-cp -R $SRC_DIR/executable/TOPAS.jar  $outdir/
+cp -R $SRC_DIR/TOPAS-1.0.1.jar  $outdir/
```

Comment:
If you use `${PKG_VERSION}`, autobump might be able to handle the next version of this.

Reply:
thanks @epruesse! Updated it in latest commit.